use-cases/bed-allocation/src/main/java/org/acme/bedallocation/domain/DepartmentSpecialism.java

```diff
@@ -8,14 +8,14 @@ public class DepartmentSpecialism {
     private String id;
 
     private Department department;
-    private Specialism specialism;
+    private String specialism;
 
     private int priority; // AKA choice
 
     public DepartmentSpecialism() {
     }
 
-    public DepartmentSpecialism(String id, Department department, Specialism specialism, int priority) {
+    public DepartmentSpecialism(String id, Department department, String specialism, int priority) {
```

Comment:
I don't understand. So we do have `DepartmentSpecialism`, but for some reason, `specialism` is still a `String` and we use it as `String` everywhere?